src/VT100.jl

```diff
@@ -413,7 +413,7 @@ function parseSGR!(em::Emulator, params)
             if f2 == 2
                 idx += 2
                 f4, f5 = params[(idx-1):idx]
-                set_cur_cell(em,Cell(cell, fg = RGB8(f3, f4, f5), flags=cell.flags | FG_IS_RGB))
+                set_cur_cell(em,Cell(cell, fg_rgb = RGB8(f3/255, f4/255, f5/255), flags=cell.flags | FG_IS_RGB))
```

Comment:
may be better to use the Ufixed constructor to avoid roundtripping through float?